hera_mc/tests/test_sys_handling.py

```diff
@@ -35,6 +35,11 @@ def test_publish_summary(self):
         msg = self.sys_h.publish_summary()
         self.assertEqual(msg, 'Not on "main"')
 
+    def test_random_update(self):
+        si = sys_handling.StationInfo()
+        si.update_stn(None)
+        si.update_arrays(None)
```

Comment:
should this test have some kind of assert? What is it testing?

Reply:
These don't return anything -- they are very internal.  This just checks the code runs.

Reply:
ok. Is there a way to test them that checks that the code does the right thing?